test/extended/testdata/builds/volumes/csi-docker-buildconfig.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: mydockertest
+  labels:
+    name: test
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: mydockerstream:latest
+  postCommit:
+    script: cat /var/run/secrets/some-secret/key
+  triggers: []
+  runPolicy: Serial
+  source:
+    dockerfile:
+      'FROM quay.io/redhat-developer/test-build-simples2i:1.2'
+  strategy:
+    type: Docker
+    dockerStrategy:
+      env:
+        - name: BUILD_LOGLEVEL
+          value: "5"
+      volumes:
+      - mounts:
+        - destinationPath: "/var/run/secrets/some-secret"
+        name: my-csi-shared-secret
+        source:
+          csi:
+            driver: csi.sharedresource.openshift.io
+            volumeAttributes:
+              sharedSecret: my-secret
```

Comment:
Per my comment to you in https://docs.google.com/document/d/1F3VFkiLUJbsUkFJcjnA2oyROYTI5RRlTJYl4ZWQoNls/edit?disco=AAAASUDEeYE add the `‘refreshResource’ attribute, and set it to a string value of false

Reply:
Another option as well, which I might like better, is that in your OCM controller PR, if you see CSI driver volume mounts with our shared resource driver here, you adjust their volume attributes to force `refreshResources` to 'false'

WDYT @adambkaplan

Reply:
@gabemontero I added the refreshResource attribute

Reply:
OK putting a comment to reflect our team discussion yesterday on this after I posted my last set of comments, namely
we are *NOT* going to have the OCM force shared resource CSI volumes to have refreshResource set to false.

That said, it is a supported option we do want to let user opt into.

So I'll need to adjust my request to you here @jkhelil ..... for each of your shared resource test items here, have one where you do not set refreshResource, and then one where you add the refreshsResource: 'false' setting

Reply:
@gabemontero what is the purpose for adding tests without refreshsResource: 'false', because we know that it shouldnt happen for builds and it will fail(correct me if i am wring about it)

Reply:
no @jkhelil both scenarios should work
- if the user does not add the `refreshResource` volume attribute, it should work
- if the user does add the `refreshResourece: 'false'` volume attribute, it should work

Hence, I want you to make sure both paths work but having 2 flavors of your various tests here.

Reply:
@gabemontero can you have a look please  to the changes regarding this last comment

Reply:
ok I'm good with the mix you have for this @jkhelil

Reply:
git the approve label from @adambkaplan and see if @coreydaley is OK with the way you added to the volumes tests he did, and we'll go from there

also, if the agnostic-cmd test is the only failure in the tests, and if you examine the tests and provide a comment to use explaining that all the sig-builds test have passed, and none of any remaining flakes are build related, we can use the skip prow command to bypass that one